Sources/Sentry/SentryCrashWrapper.m

```diff
@@ -76,6 +76,11 @@ - (uint64_t)freeMemory
     return sentrycrashcm_system_freememory();
 }
 
+- (uint64_t)freeStorage
```

Comment:
l: can we rename this to 'freeStorageSize'?
I was confused the first time I read this function. Same for the C function.

Reply:
And `freeMemory` was not confusing? I'd rather keep it consistent with existing properties.